00:00:00.008123 main     VirtualBox COM Server 6.1.38 r153438 win.amd64 (Sep  1 2022 14:20:23) release log
00:00:00.008125 main     Log opened 2022-09-24T09:06:37.835113400Z
00:00:00.008125 main     Build Type: release
00:00:00.008129 main     OS Product: Windows 11
00:00:00.008130 main     OS Release: 10.0.22621
00:00:00.008130 main     OS Service Pack: 
00:00:00.083084 main     DMI Product Name: HP Pavilion Gaming Laptop 15-ec1xxx
00:00:00.094641 main     DMI Product Version:  
00:00:00.094650 main     Firmware type: UEFI
00:00:00.094886 main     Secure Boot: VERR_PRIVILEGE_NOT_HELD
00:00:00.095941 main     Host RAM: 7541MB (7.3GB) total, 1142MB available
00:00:00.095946 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.095947 main     Process ID: 13728
00:00:00.095947 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.217476          VirtualBox: object creation starts
00:00:00.218452          Home directory: 'C:\Users\AMIT\.VirtualBox'
00:00:00.218986          Installed Drivers:
00:00:00.222478            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.38.3438)
00:00:00.223892            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.38.3438)
00:00:00.224881            C:\WINDOWS\system32\DRIVERS\VBoxSup.sys (Version: 6.1.38.3438)
00:00:00.228692            C:\WINDOWS\System32\drivers\VBoxNetAdp6.sys (Version: 6.1.38.3438)
00:00:00.241203          Loading settings file "C:\Users\AMIT\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.247525          usbLibGetDevices: Starting USB device enumeration
00:00:00.247731          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&226545ef&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 4
00:00:00.248237          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&9c5c6e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.249236          usbLibGetDevices: Found 2 USB devices, 0 captured
00:00:00.250540          USB: Unknown USB device detected (idVendor: 0x0408, idProduct: 0x5365)
00:00:00.251402          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0029)
00:00:00.360586          netIfIsWireless: CreateFile on '\\.\{3468485A-8272-4623-8675-7439EF919931}' failed with rcWin=2 (0x2) - ignoring
00:00:00.365590          HostDnsMonitor: initializing
00:00:00.373891          HostDnsMonitor: updating information
00:00:00.374213          HostDnsMonitor: old information
00:00:00.374214            no server entries
00:00:00.374215            no domain set
00:00:00.374216            no search string entries
00:00:00.374216          HostDnsMonitor: new information
00:00:00.374216            server 1: 192.168.1.1
00:00:00.374217            domain: bbrouter
00:00:00.374218            no search string entries
00:00:00.409411          VD: VDInit finished with VINF_SUCCESS
00:00:00.422633          Loading settings file "C:\Users\AMIT\VirtualBox VMs\CentOS_7.9.2009_VBG_LinuxVMImages.COM\CentOS_7.9.2009_VBG_LinuxVMImages.COM.vbox" with version "1.16-windows"
00:00:00.427582          VirtualBox: object created
00:00:03.236162          Saving settings file "C:\Users\AMIT\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:03.244133          Saving settings file "C:\Users\AMIT\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:08.553863          Launched VM: 61267712 pid: 15860 (0x3df4) frontend: GUI/Qt name: CentOS_7.9.2009_VBG_LinuxVMImages.COM
00:00:13.837923          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={70401eef-c8e9-466b-9660-45cb3e9979e4} aComponent={ExtPackManagerWrap} aText={No extension pack by the name 'Oracle VM VirtualBox Extension Pack' was found}, preserve=false aResultDetail=0
00:00:19.361005          Saving settings file "C:\Users\AMIT\VirtualBox VMs\CentOS_7.9.2009_VBG_LinuxVMImages.COM\CentOS_7.9.2009_VBG_LinuxVMImages.COM.vbox" with version "1.16-windows"
00:00:19.569909          netIfIsWireless: CreateFile on '\\.\{3468485A-8272-4623-8675-7439EF919931}' failed with rcWin=2 (0x2) - ignoring
00:00:26.989201          Saving settings file "C:\Users\AMIT\VirtualBox VMs\CentOS_7.9.2009_VBG_LinuxVMImages.COM\CentOS_7.9.2009_VBG_LinuxVMImages.COM.vbox" with version "1.16-windows"
00:13:35.024891 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:13:35.025181 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&226545ef&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 4
00:13:35.025424 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&9c5c6e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:13:35.025821 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
01:50:41.485572 USBPROXY usbLibGetDevices: Starting USB device enumeration
01:50:41.487913 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&226545ef&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 4
01:50:41.489272 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&9c5c6e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
01:50:41.492897 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
02:00:27.424478 USBPROXY usbLibGetDevices: Starting USB device enumeration
02:00:27.424806 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&226545ef&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 4
02:00:27.425036 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&9c5c6e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
02:00:27.425234 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
02:10:10.326000 MainPower Host suspending: Paused 1 VMs
02:10:11.060469 USBPROXY usbLibGetDevices: Starting USB device enumeration
02:10:11.060831 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&226545ef&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 4
02:10:11.061141 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&9c5c6e&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
02:10:11.061412 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
02:28:52.361497 MainPower Host resumed: Resumed 1 VMs
02:28:54.313735 dns-monitor HostDnsMonitor: updating information
02:28:54.313809 dns-monitor HostDnsMonitor: unchanged
02:36:58.301503          Saving settings file "C:\Users\AMIT\VirtualBox VMs\CentOS_7.9.2009_VBG_LinuxVMImages.COM\CentOS_7.9.2009_VBG_LinuxVMImages.COM.vbox" with version "1.16-windows"
02:36:58.323074          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={MachineWrap} aText={Saved screenshot thumbnail data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
02:36:58.335115          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={MachineWrap} aText={Saved screenshot thumbnail data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
02:36:58.634502 SaveState Saving settings file "C:\Users\AMIT\VirtualBox VMs\CentOS_7.9.2009_VBG_LinuxVMImages.COM\CentOS_7.9.2009_VBG_LinuxVMImages.COM.vbox" with version "1.16-windows"
02:37:03.164037          Saving settings file "C:\Users\AMIT\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
02:37:03.174934          Saving settings file "C:\Users\AMIT\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
02:37:08.748344 main     VirtualBox: object deletion starts
02:37:08.749213 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'C:\Users\AMIT\VirtualBox VMs\CentOS_7.9.2009_VBG_LinuxVMImages.COM\Snapshots/{705eb074-20c2-4fa2-aee5-d10d736867b5}.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
02:37:08.749287 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'C:\Users\AMIT\VirtualBox VMs\CentOS_7.9.2009_VBG_LinuxVMImages.COM\CentOS_7.9.2009_VBG_LinuxVMImages.COM-disk001.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
02:37:08.752492 main     HostDnsMonitor: shutting down ...
02:37:08.752581 main     HostDnsMonitor: shut down
02:37:08.755956 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
02:37:08.756895 main     VirtualBox: object deleted
